.github/workflows/regression-report.yml

```diff
@@ -30,6 +30,22 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  check_pr_status:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR status
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_URL: ${{ github.event.issue.pull_request.url }}
+        run: |
+          PR_STATE=$(curl -s -H "Authorization: token $GITHUB_TOKEN" \
+                     -H "Accept: application/vnd.github.v3+json" \
+                     "$PR_URL" | jq -r .state)
+          if [ "$PR_STATE" != "open" ]; then
+            echo "PR is not active"
+            exit 1
+          fi
+
   parse_comment:
```

Comment:
Parse_comment should require `check_pr_status`
So it  will not execute if check_pr_status failing

Reply:
done